puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -3,6 +3,11 @@
     cwd => '/',
   }
 
+  class { 'postgresql::globals':
+    manage_package_repo => true,
+    version             => '9.6',
```

Comment:
Why 9.6 in particular? Shouldn't we try with the OS versions since that's what we deploy on?

Reply:
That's 9.2 with CentOS 7, while later versions do have some performance improvements.